code/datums/components/mood.dm

```diff
@@ -43,7 +43,7 @@
 
 /datum/component/mood/proc/print_mood(mob/user)
 	var/msg = "<span class='info'>*---------*\n<EM>Your current mood</EM></span>\n"
```

Comment:
While we're at it, can we change this "Your" to a "My" or something? Everything else looks to be in first person, but this is in second person.

Actually, why not replace this line with "Your current mental status:", since "mood" is the name of a specific submechanic of your mental status (and is detailed in a later line)?

Reply:
```suggestion
	var/msg = "<span class='info'>*---------*\n<EM>My current mental status:</EM></span>\n"
```